userguide/tutorials/plugin_development.adoc

```diff
@@ -180,6 +180,23 @@ In addition to the classes listed above, some of the other classes in the `hello
 * https://github.com/killbill/killbill-hello-world-java-plugin/blob/3aa938d19fdfba81c7c035b45c3f17cac74db177/src/main/java/org/killbill/billing/plugin/helloworld/HelloWorldConfigurationHandler.java[_HelloWorldConfigurationHandler_]: Most plugins require custom configuration. A configuration handler similar to the `HelloWorldConfigurationHandler` can be used to read the configuration properties.
 * https://github.com/killbill/killbill-hello-world-java-plugin/blob/3aa938d19fdfba81c7c035b45c3f17cac74db177/src/main/java/org/killbill/billing/plugin/helloworld/HelloWorldHealthcheck.java[_HelloWorldHealthcheck_] and https://github.com/killbill/killbill-hello-world-java-plugin/blob/3aa938d19fdfba81c7c035b45c3f17cac74db177/src/main/java/org/killbill/billing/plugin/helloworld/HelloWorldHealthcheckServlet.java[_HelloWorldHealthcheckServlet_]: Can be used to provide the health status of the plugin.
 
+== Enabling Per-tenant Configuration
+
+As explained https://docs.killbill.io/latest/plugin_installation.html#per-tenant-configuration[_here_], Kill Bill supports per tenant configuration for plugins. In order enable per tenant configuration, the following needs to be done:
+
+. Create a custom configuration handler similar to https://github.com/killbill/killbill-hello-world-java-plugin/blob/3aa938d19fdfba81c7c035b45c3f17cac74db177/src/main/java/org/killbill/billing/plugin/helloworld/HelloWorldConfigurationHandler.java[_HelloWorldConfigurationHandler_]
+
+. Initialize it in the `start` method of your activator class as follows (See https://github.com/killbill/killbill-hello-world-java-plugin/blob/3aa938d19fdfba81c7c035b45c3f17cac74db177/src/main/java/org/killbill/billing/plugin/helloworld/HelloWorldActivator.java#L57[HelloWorldActivator.start]):
++
+[source, java]
+----
+helloWorldConfigurationHandler = new HelloWorldConfigurationHandler(region, PLUGIN_NAME, killbillAPI);
+final Properties globalConfiguration = helloWorldConfigurationHandler.createConfigurable(configProperties.getProperties());
+helloWorldConfigurationHandler.setDefaultConfigurable(globalConfiguration);
+----
+
+Now, you can use the `ConfigurationHandler` to retrieve per-tenant properties. You can see an example of this in the <<retrieve_configuration_example, Authentication Steps>> section below.
+
 == Setting up a Breakpoint and Debugging
```

Comment:
```suggestion
== Setting Up a Breakpoint and Debugging
```

Reply:
^^https://capitalizemytitle.com/what-is-a-phrasal-verb/

I am terrible at remembering what to capitalize, so I frequently use the Capitalize My Title website!

Reply:
Thanks for pointing me to [https://capitalizemytitle.com](https://capitalizemytitle.com/what-is-a-phrasal-verb/)  👍